core/modules/image/config/image.style.card.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_config_name": "image.style.card",
+    "storage": 4,
```

Comment:
Can you change this to 1 (`IMAGE_STORAGE_NORMAL`), and move this entire file to within `profiles/standard/config`? If cards are only provided by the standard profile, this image style should only exist when using it.